docs/application/dotnet/guides/nui/visuals.md

```diff
@@ -1,390 +1,552 @@
+<a name="visualMap"></a>
+## VisualMap encapsulates common properties of Visuals 
 
-    ```
-    var colorMap = new ColorVisual{Color=Color.White;};
-    var _colorVisual = VisualFactory.Instance.CreateVisual(colorMap.OutputVisualMap);
-    RegisterVisual(ColorVisualPropertyIndex, _colorVisual);
-    ```
+The VisualMap is a base class for all Visuals.
 
-    You can also create a visual using an output map in a control property. The following example uses the `Background` property:
+**Table: The VisualMap properties - all are optional.**
 
-    ```
-    ColorVisual colorVisualMap1 = new ColorVisual();
-    colorVisualMap1.Color = Color.Green;
-    _visualView.Background = colorVisualMap1.OutputVisualMap;
+| Property            | Type                        | Default Value                    | Description                                                                                                       |
+|---------------------|-----------------------------|----------------------------------|-------------------------------------------------------------------------------------------------------------------|
+| `Size`              | `Size2D`                    | `(1, 1)`                         | The Visual size - depending on the `SizePolicy` value is interpreted as the relative or the absolute size.        |
+| `RelativeSize`      | `RelativeVector2`           | `(1.0f, 1.0f)`                   | The Visual size - use to specify the relative size as a percentage of the parent.                                 |
+| `SizePolicy`<br>`SizePolicyWidth`<br>`SizePolicyHeight` | `VisualTransformPolicyType` | `Relative`   | Specifies whether the Visual size/width/height is relative (`Relative`) or absolute (`Absolute`). |
+| `Position`          | `Vector2`                   | `(0.0f, 0.0f)`                   | The Visual's offset (absolute or relative - percentage) specified as the shift of the Visual reference point with respect to the parent's reference point (values on x and y axes increase towards right and bottom, respectively). |
+| `RelativePosition`  | `RelativeVector2`           | `(0.0f, 0.0f)`                   | The Visual's offset specified as a percentage of the parent.                                                      |
+| `PositionPolicy`<br>`PositionPolicyX`<br>`PositionPolicyY` | `VisualTransformPolicyType` | `Relative`   | Specifies whether the x and/or y offsets are relative (`Relative`) or absolute (`Absolute`).   |
+| `Origin`            | `Visual.AlignType`          | `Center`                         | Specifies the parent's reference point (possible values: `TopBegin`, `TopCenter`, `TopEnd`, `CenterBegin`, `Center`, `CenterEnd`, `BottomBegin`, `BottomCenter`, `BottomEnd`). |
+| `AnchorPoint`       | `Visual.AlignType`          | `Center`                         | Specifies the Visual's reference point used e.g. to define the Visual position (possible values as for the `Origin`).  |
```

Comment:
Origin specifies the point on the parent's plane (x, y or width/height). The visual will be anchored to this point.
Because the visual is a 2d object, we need to determine a point on its surface, which will be our reference point. We call this the anchor point.

it is like ParentOrigin and PivotPoint for a View (https://docs.tizen.org/application/dotnet/guides/nui/view/).
Do you want me to add a sketch similar to this one? :
https://docs.tizen.org/application/dotnet/guides/nui/media/view_Figure_Position.png

Reply:
please check now - I added a picture